invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/base.js

```diff
@@ -115,26 +115,60 @@ export const DashboardSearchLayoutHOC = ({
   newBtn = () => null,
   ...props
 }) => {
-  const DashboardUploadsSearchLayout = (props) => (
-    <Container>
+
+  const DashboardUploadsSearchLayout = (props) => {
+    const closeSidebarBtnRef = React.useRef();
+    const [sidebarVisible, setSidebarVisible] = React.useState(false);
+
+    return (
       <Grid>
-        <Grid.Row columns={3}>
-          <Grid.Column width={4} />
-          <Grid.Column width={8}>
-            <SearchBar placeholder={searchBarPlaceholder} />
-          </Grid.Column>
-          <Grid.Column width={4}>{newBtn}</Grid.Column>
-        </Grid.Row>
+        <Grid.Column only="mobile tablet" mobile={2} tablet={1}>
+          <Button basic icon="sliders"
+                  onClick={() => setSidebarVisible(true)}
+                  aria-label={i18next.t("Filter results")}
+          />
+        </Grid.Column>
+
+        <Grid.Column only="computer" width={4}/>
```

Comment:
is it possible to add "right aligned" to the next column instead of having this one as negative space ?